.github/workflows/style.yaml

```diff
@@ -0,0 +1,73 @@
+on:
+  push:
+    branches:
+      - devel
+      - RELEASE_**
```

Comment:
This seems to run when pushing to devel branch (without PR)?

Can this cause any issues? I see it little bit concerning if GHA can modify our "main" (in this case branch called devel) branch. Everything that is merged into "main" should be accepted by us, and GHA should not be able to modify our main code base. Moreover, this can cause problems in the future if the GHA is modified in the future, and this issue is not recognized.

I would be more comfortable, if GHA would have access only to PRs (anyways everything should go through it).